rke/cis_config.go

```diff
@@ -0,0 +1,27 @@
+package rke
+
+import (
+	v3 "github.com/rancher/types/apis/management.cattle.io/v3"
+)
+
+func loadCisDefaultConfigs() map[string]v3.CisDefaultConfig {
+	return map[string]v3.CisDefaultConfig{
+		"default": {
+			BenchmarkVersion: "rke-cis-1.4",
+		},
+		"v1.15": {
+			BenchmarkVersion: "rke-cis-1.4",
+		},
+	}
+}
+
+func loadCisBenchmarkVersionInfo() map[string]v3.CisBenchmarkVersionInfo {
+	return map[string]v3.CisBenchmarkVersionInfo{
+		"rke-cis-1.4": {
+			MinKubernetesVersion: "1.13",
+		},
+		"cis-1.4": {
```

Comment:
what's the difference between `cis-1.4` and `rke-cis-1.4`?

Reply:
cis-1.4 is directly upstream, rke-cis-1.4 is customized for RKE clusters.

Reply:
The upstream tests don't work on RKE clusters as the tool doesn't understand running k8s components in clusters.

Reply:
I see, why do we include upstream `cis-1.4` in metadata then?

Reply:
in case the user wants to run the upstream kube-bench's version cis-1.4 directly

Reply:
I don't understand, thought CIS was only for RKE clusters?

Reply:
The upstream tool comes with tests in `cis-1.4` folder which can be run on any cluster, but the tool would report some false negative errors for RKE clusters as the tool/tests do not understand rke. Hence we use tests from `rke-cis-1.4` with mods for making the tests compatible with RKE clusters.